src/App.tsx

```diff
@@ -1,61 +1,125 @@
+import { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import { TodoList } from './components/TodoList';
+import todosFromServer from './api/todos';
+import { Todo } from './types/Todo';
+
+const currentTodos: Todo[] = todosFromServer.map(todo => {
+  const newProperty = usersFromServer.find(user => user.id === todo.userId);
+
+  return {
+    ...todo,
+    email: newProperty?.email || '',
+    name: newProperty?.name || '',
+  };
+});
```

Comment:
Consider adding error handling for cases where `usersFromServer` or `todosFromServer` might be undefined or null. This will prevent potential runtime errors if the data is not loaded correctly.